### PR TITLE
god damn empty bash variable

### PR DESCRIPTION
**Pull request by geoffjentry**
Strictly speaking the only issue here was the unset config file var messing up the java command. However the rest of this are little things which I found to be problematic and/or potentially troublesome while messing around

### CODE REVIEW

File: test_cromwell.sh [hunk 1]
```diff
@@ -86,13 +86,17 @@ if [[ -n ${CROMWELL_BRANCH} ]]; then
...
-    sbt clean assembly >> ../${ASSEMBLY_LOG} 2>&1
+    sbt clean assembly >> "${ASSEMBLY_LOG}" 2>&1
     cd ..
-    CROMWELL_JAR="cromwell/target/scala-2.11/cromwell-*.jar"
+    CROMWELL_JAR=`find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar"`
 fi
 
 echo "Starting Cromwell, jar is ${CROMWELL_JAR}"
-java "${CONFIG_STRING}" -jar "${CROMWELL_JAR}" server >> "${CROMWELL_LOG}" 2>&1 &
+if [ -n "$CONFIG_STRING" ]; then
```
Comment (ruchim) [inline test_cromwell.sh:95]: Is Travis not going to use the config? or this is for just development purposes?

**Comment by ruchim** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/101/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: test_cromwell.sh (hunk 1 above)
Comment (geoffjentry) [reply test_cromwell.sh:95]: jes uses it, but more generally this script could be used by anything and may or may not have a config. local is just using application.conf

File: test_cromwell.sh [hunk 2]
```diff
@@ -86,13 +86,17 @@ if [[ -n ${CROMWELL_BRANCH} ]]; then
     git checkout "${CROMWELL_BRANCH}"
     git pull
     echo "Building Cromwell"
-    sbt clean assembly >> ../${ASSEMBLY_LOG} 2>&1
+    sbt clean assembly >> "${ASSEMBLY_LOG}" 2>&1
     cd ..
-    CROMWELL_JAR="cromwell/target/scala-2.11/cromwell-*.jar"
+    CROMWELL_JAR=`find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar"`
```
Comment (kshakir) [inline test_cromwell.sh]: Switch backticks to `$()`

``` bash
CROMWELL_JAR=$(find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar")
```

Comment (geoffjentry) [reply test_cromwell.sh]: Heh i saw that in the bash linter but at that point i was so frustrated i didn't want to rock the boat. I'll try it and make sure it doesn't break anything

**Comment by geoffjentry** [comment]
Realized this wasn't tested against the JES fork in Travis, will do so prior to merge

**Comment by kshakir** [comment]
There are a couple of other warnings by [shellcheck](https://www.shellcheck.net/) (one including quoted variables 😉 ), but nothing major. 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/101/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by geoffjentry** [comment]
@kshakir one in particular I remembered from the first time around, the suggestion didn't work correctly. I'll run it through again since I'm fresh today and slightly less irritated at the whole affair ;)

**Comment by geoffjentry** [comment]
Actually I believe both of the remaining warnings were things which never worked previously when I tried applying them. They look familiar.

**Pull request merged**